e4s_cl/logger.py

```diff
@@ -451,6 +452,22 @@ def add_file_handler(
 
     return True
 
+def symlink_latest():
+    if os.path.exists(Path(LOG_FILE.parent, LOG_ID)):
+        try:
+            os.unlink(LOG_LATEST)
+        except OSError as err:
+            _ROOT_LOGGER.debug("Unlink %s failed: %s", LOG_LATEST.as_posix(),
+                               str(err))
+
+        try:
+            os.symlink(Path(LOG_FILE.parent, LOG_ID), LOG_LATEST)
+        except OSError as err:
+            _ROOT_LOGGER.debug("Symlink %s failed: %s", LOG_LATEST.as_posix(),
+                               str(err))
+        else:
+            os.environ[LOG_ID_MARKER] = LOG_LATEST.name
```

Comment:
Delete this as it becomes irrelevant when moved to the end of execution

Reply:
Done